yarn-project/end-to-end/src/e2e_p2p_network.test.ts

```diff
@@ -47,7 +48,13 @@ describe('e2e_p2p_network', () => {
   let deployL1ContractsValues: DeployL1Contracts;
 
   beforeEach(async () => {
-    ({ teardown, config, logger, deployL1ContractsValues } = await setup(0));
+    // If we want to test with interval mining, we can use the local host and start `anvil --block-time 12`
+    const useLocalHost = false;
```

Comment:
could have the interval mining settings added to the createAnvilOpts instead? So it is not as siloed to this test

Reply:
Was not always for interval, sometimes also just used it to get proper errors back. For cases like the `e2e_token_contract` I think it might cause quite a bit of pain if they are all using the same local anvil for example (unless interval mining ye).